tests/test_regression.py

```diff
@@ -43,17 +43,17 @@ def test_sbottom_regionA_1300_205_60(
     CLs_obs, CLs_exp = calculate_CLs(
         sbottom_regionA_bkgonly_json, sbottom_regionA_1300_205_60_patch_json
     )
-    assert CLs_obs == pytest.approx(0.24443627759085326, rel=1e-5)
+    assert CLs_obs == pytest.approx(0.2444418, rel=1e-5)
     assert np.all(
         np.isclose(
             np.array(CLs_exp),
             np.array(
                 [
-                    0.09022509053507759,
-                    0.1937839194960632,
-                    0.38432344933992,
-                    0.6557757334303531,
-                    0.8910420971601081,
+                    np.array(0.09023417),
```

Comment:
why are these wrapped in nested arrays?

Reply:
We made changes in the return structure a while back so that `pyhf.infer.hypotest(..., return_expected_set=True)` returns a list of tensors. The goal he was to actually match that return structure, and as these are all being run with the NumPy backend only I thought just use `np.array`.

```
>>> import pyhf
>>> model = pyhf.simplemodels.hepdata_like(signal_data=[12.0, 11.0], bkg_data=[50.0, 52.0], bkg_uncerts=[3.0, 7.0])
>>> data = [51, 48] + model.config.auxdata
>>> test_mu = 1.0
>>> CLs_obs, CLs_exp = pyhf.infer.hypotest(test_mu, data, model, qtilde=True, return_expected_set=True)
>>> CLs_obs
array(0.0525151)
>>> CLs_exp
[array(0.0026063), array(0.01382019), array(0.06445367), array(0.2352575), array(0.5730375)]
```

This change is not strictly needed, but it seemed like it should be dealt with here if anywhere.